Assets/Scripts/Scenes/InGame/Enemy/Trace/Feature/FeatureView.cs

```diff
@@ -5,32 +5,41 @@
 using System;
 using Scenes.Ingame.Stage;
 using System.Collections;
+using EPOOutline;
 
 namespace Scenes.Ingame.Enemy.Trace.Feature
 {
     public class FeatureView : MonoBehaviour
     {
         GameObject _enemy;//TODO:敵が複数体だった場合の対応
         private const float RANGE = 5;
+        private const float NEAR_RANGE = 1;
         private GameObject[] _stageInteracts;
         private GameObject[] nearStageObject = null;
+        private GameObject[] moreNearStageObject = null;
         private GameObject interactTarget = null;
+        private GameObject nearInteractTarget = null;
         private GameObject _floor;
         private Subject<Unit> _onDestroy = new Subject<Unit>();
         public IObservable<Unit> onDestroy { get => _onDestroy; }
         private ReactiveProperty<StageTile> _stageTile = new ReactiveProperty<StageTile>();
         public IObservable<StageTile> OnStageTileChange { get { return _stageTile; } }
         private Vector3 direction = new Vector3(0, -1, 0);
+        private Vector3 direction_wall = new Vector3(1, 0, 0);
         [SerializeField] private GameObject _trackSprite;
-
+        private int randomValue;
+        private int traceSelect;
+        private int _openInterval = 5;
+        private int _maxTraceType = 4;
```

Comment:
```suggestion
        private int _maxTraceType = 2;
```
現状switch文内では0か1しか使っていないので、_maxTraceType の値は2が適切だと思います。